Bottom: 3bf244200670464f8471701b5581e84f26e43d39
Top:    957a7f1baea672141b12601bb89475d8421d88f2
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-10-21 22:21:19 +0800

Refresh of vhost_net-vmapped-metadata

---

diff --git a/mm/vmalloc.c b/mm/vmalloc.c
index 3d2d369..8a43db6 100644
--- a/mm/vmalloc.c
+++ b/mm/vmalloc.c
@@ -1651,9 +1651,10 @@ void *vmap(struct page **pages, unsigned int count,
 	if (!area)
 		return NULL;
 
-	if (map_vm_area(area, prot, pages))
+	if (map_vm_area(area, prot, pages)) {
 		vunmap(area->addr);
 		return NULL;
+	}
 
 	return area->addr;
 }
